packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -252,17 +252,21 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
     <div ref={containerRef} className={mergeStyles(containerDivStyles)}>
       <Stack verticalFill grow styles={compositeOuterContainerStyles} id={compositeParentDivId}>
         <Stack horizontal grow>
-          {!isMobileWithActivePane && (
-            <Stack.Item grow styles={callCompositeContainerStyles}>
-              <CallComposite
-                {...props}
-                formFactor={formFactor}
-                options={{ callControls: false }}
-                adapter={callAdapter}
-                fluentTheme={fluentTheme}
-              />
-            </Stack.Item>
-          )}
+          <Stack.Item
+            grow
+            styles={callCompositeContainerStyles}
+            // Perf: Instead of removing the video gallery from DOM, we hide it to prevent re-renders.
+            style={{ display: isMobileWithActivePane ? 'none' : 'flex' }}
```

Comment:
I don't know with the style prop if it has to be memoized... this will be a new object each render so maybe?

Reply:
We have had this discussion before. I think the verdict was to memoize `styles` and small inline `style` were fine.
Since `styles` are a prop for underlying fluent components and cause re-renders while `style` is just css.

Reply:
Actually, I am still gonna memoize it and move it outside. Good to follow a standard practice I think.
Thanks for pointing it out